src/services/checkin/checkin-router.ts

```diff
@@ -0,0 +1,46 @@
+import { Router } from "express";
+import { StatusCodes } from "http-status-codes";
+import { ScanValidator } from "./checkin-schema";
+import { Database } from "../../database";
+import RoleChecker from "../../middleware/role-checker";
+import { Role } from "../auth/auth-models";
+// import dotenv from "dotenv";
+import { validateQrHash } from "../attendee/attendee-utils";
+import { checkInUserToEvent } from "./checkin-utils";
+
+const adminRouter = Router();
+
+adminRouter.post(
+    "/checkin/scan/staff",
```

Comment:
remove /checkin here
```suggestion
    "/scan/staff",
```